webapp/src/main/java/life/qbic/events/EventStore.java

```diff
@@ -0,0 +1,37 @@
+package life.qbic.events;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Implementation of a basic event store. It handles events and provides accessor methods to retain
+ * the events later.
+ */
+public class EventStore {
+
+  private final EventRepository eventRepository;
+
+  public EventStore(EventRepository eventRepository) {
+    this.eventRepository = eventRepository;
+  }
```

Comment:
Effective Java: Favour static factory methods over constructors for object creation.  This also works when injecting dependencies. 

```suggestion
  public EventStore(EventRepository eventRepository) {
    this.eventRepository = eventRepository;
  }
```

Reply:
Hi @sven1103, you suggested earlier to make the EventStore a singleton. This would mean introducing a `getInstance()` method. Do you know how one can implement a singleton with different repositories or is there only one repository allowed? I implemented a static `getInstance()`. Is this what you had in mind?

Reply:
Hi @KochTobi i think a singleton is fine for now, no need for multiple EventStores at the moment. when the model requires it, we change it :)